pkg/crc/version/version.go

```diff
@@ -39,3 +57,37 @@ func GetCommitSha() string {
 func GetBundleVersion() string {
 	return bundleVersion
 }
+
+func getCRCLatestVersionFromMirror() (*semver.Version, error) {
+	var releaseInfo CrcReleaseInfo
+	response, err := http.Get(releaseInfoLink)
+	if err != nil {
+		return &semver.Version{}, err
+	}
+	releaseMetaData, err := ioutil.ReadAll(response.Body)
+	if err != nil {
+		return &semver.Version{}, err
+	}
+	response.Body.Close()
```

Comment:
shouldn't this be like `defer response.Body.Close()` so that if you hit the error during getting the page you should also close it?

Reply:
Well, we are already checking the error before the close, and if no errors encountered, then only we close, otherwise we'd try to close on a null.

Reply:
Check https://golang.org/pkg/net/http/ once second section, I think as a better practice you should always close the connection.

Reply:
`defer` is always the better option here.

Reply:
Right, that uses `defer` to close, why i went without this is, i didn't want the start command to panic, suppose the `Get` method returns an error, since using `defer` would call the `.Close()` func on `response` all the time, even when its null we might get into a panic.